golem-worker-service-base/src/gateway_execution/http_handler_binding_handler.rs

```diff
@@ -27,15 +28,13 @@ use std::convert::Infallible;
 use std::str::FromStr;
 use std::sync::Arc;
 
-use super::{GatewayWorkerRequestExecutor, WorkerRequestExecutorError};
-
 #[async_trait]
 pub trait HttpHandlerBindingHandler<Namespace> {
     async fn handle_http_handler_binding(
         &self,
         namespace: &Namespace,
         worker_detail: &WorkerDetail,
-        request_details: &HttpRequestDetails,
+        incoming_http_request: IncomingHttpRequest,
```

Comment:
Note that this is the wasi / golem-wit incoming-request, not anything http specific